helpers/event_helper.rb

```diff
@@ -3,9 +3,15 @@ def event_time(event_start, event_end = nil, mobile_only: false)
     klass = "u-mobile" if mobile_only
 
     content_tag :div, class: klass do
-      content_tag :p, class: "Event-time #{klass}" do
+      content_tag :p, class: "Event-time Text base blue alternative" do
         event_end ? "#{event_start} > #{event_end}" : event_start
       end
     end
   end
+
+  def link_to_event(name, text, **args)
+    link_to "/schedule##{speaker_id(name)}", args do
+      text || "TBA"
```

Comment:
if this is just text, no need to have it as a block

also, no point in swapping the arguments of this method comparing to the original `link_to`. the interface should be `link_to_event(text, name, **args)`